sys-kernel/scx/scx-1.0.5.ebuild

```diff
@@ -319,14 +319,14 @@ DEPEND="
 "
 RDEPEND="
 	${DEPEND}
-	dev-util/bpftool
 "
 BDEPEND="
 	app-misc/jq
 	$(llvm_gen_dep '
 		sys-devel/clang:${LLVM_SLOT}=[llvm_targets_BPF(-)]
 	')
 	>=virtual/rust-1.72
+	<=dev-util/bpftool-7.5.0-r1
```

Comment:
Please do <dev-util/bpftool-7.5.0-r2 instead, but also, this needs a bug reference and an explanation.

Reply:
See my comment on bgo. IMHO this is even more wrong since everybody should depend on >=bpftool-7.5.0, not the old 6.5 versions.